app/views/management.py

```diff
@@ -1,17 +1,64 @@
-from flask import Blueprint, render_template
+from flask import Blueprint, render_template, current_app, request, jsonify
 from app.helpers import login_required
+from app.db import get_db
 
 management = Blueprint("management", __name__)
 
 
-@management.route("/inventory")
+@management.route("/management")
+@login_required
+def management_layout():
+    return render_template("management/management_layout.html")
+
+
+@management.route("/products")
+@login_required
+def products():
+    return render_template("management/products.html")
+
+
+@management.route("/search")
+@login_required
+def search():
+    database = get_db()
+    q = request.args.get("q")
+
+    Json = []
+    if q:
+        items = database.execute(
+            "SELECT name FROM items WHERE name LIKE ?||?", (q, "%")
```

Comment:
Should apply standard format for queries:
"""
SELECT name
FROM items
WHERE name
LIKE ?||?
"""